opendevin/runtime/docker/ssh_box.py

```diff
@@ -731,6 +730,8 @@ def restart_docker_container(self):
 
             # start the container
             logger.info(f'Mounting volumes: {self.volumes}')
+            # update the container image
+            self.docker_client.images.pull(self.container_image)
```

Comment:
This is within `restart_docker_container`, which itself is called in a loop in `__init__`.
Might be cleaner to do the pull outside of the loop, just before the `while n_tries > 0:`?